volatility3/framework/symbols/windows/extensions/mft.py

```diff
@@ -5,14 +5,20 @@
 import logging
 from typing import Dict, Iterator, List, Optional, Tuple
 
+from volatility3 import framework
 from volatility3.framework import constants, exceptions, interfaces, objects
 
 vollog = logging.getLogger(__name__)
 
 
-class MFTEntry(objects.StructType):
+class MFTEntry(objects.StructType, interfaces.configuration.VersionableInterface):
```

Comment:
I'm not sure this is a good plan, since they're applied to the symbol tables by the framework, you may get handed one and have no idea which version it is (and checking it after the fact will be problematic)...  5:S

Reply:
Ok, reverted